packages/wonder-blocks-tooltip/components/tooltip-tail.js

```diff
@@ -273,38 +288,51 @@ export default class TooltipArrow extends React.Component<Props> {
     }
 }
 
+/**
+ * Some constants to make styles easier to understand.
+ * NOTE: The widths and heights refer to the downward-pointing tail
+ * (i.e. placement="top"). When the tail points to the left or right instead,
+ * the width/height are inverted.
+ */
+const minDistanceFromCorners = Spacing.medium;
+const distanceFromAnchor = Spacing.xSmall;
+
+const arrowWidth = Spacing.large;
+const arrowHeight = Spacing.small;
+
+const fullTailWidth = arrowWidth + 2 * minDistanceFromCorners;
```

Comment:
What happens if this is greater than the available space on that edge?

Reply:
That doesn't happen. The bubble is sized to its container (flex) and the container is sized to the arrow and bubble contents...so it all sizes properly.